### ISSUE TITLE
intellij idea run configuration Terasology PC fails to find resources

### ISSUE THREAD

**Issue by keturn** [MovingBlocks/Terasology#3951]
### What you were trying to do

Run _Terasology PC_ from IntelliJ.

### What actually happened

Terasology quickly crashes when engine.Terasology.configureSplashScreen gets nulls in response to its `getResource` queries.

### How to reproduce

* gradlew clean cleanIdea idea game
* …quit the game, that was just to demonstrate it does run from the gradle task.
* Open in IntelliJ, run _Terasology PC_

### Log details and game version

git bd162d4c9b6f6c61997c782daee5104e28576d4a

```
Exception in thread "main" java.lang.IllegalArgumentException: input == null!
	at java.desktop/javax.imageio.ImageIO.read(ImageIO.java:1400)
	at org.terasology.splash.overlay.ImageOverlay.<init>(ImageOverlay.java:38)
	at org.terasology.splash.overlay.TriggerImageOverlay.<init>(TriggerImageOverlay.java:30)
	at org.terasology.engine.Terasology.configureSplashScreen(Terasology.java:216)
	at org.terasology.engine.Terasology.main(Terasology.java:134)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at com.intellij.rt.execution.application.AppMainV2.main(AppMainV2.java:128)
```

### Computer details

:penguin:

**Comment by keturn** [comment]
I think I found the culprit, but I haven't figured out yet how this translates back to the gradle task that makes the run configs.

![PC-Paths](https://user-images.githubusercontent.com/83819/81631029-d3373700-93bb-11ea-98de-d880f36b52e1.png)
![PC-Sources-before](https://user-images.githubusercontent.com/83819/81631038-d8948180-93bb-11ea-980e-c214cfdacea0.png)
![PC-Sources-after](https://user-images.githubusercontent.com/83819/81631046-ddf1cc00-93bb-11ea-9f9e-a9a870065abf.png)

The PC "module" (wow that word is just getting more overloaded) excludes build outputs from its paths, but the resources it needs are in there. That path is present in the classpath of the gradle `PC:game` task.

**Comment by keturn** [comment]
I guess that "exclude output paths" on the **Paths** tab is just talking about editor-exclusion, not runtime-exclusion. [Help](http://127.0.0.1:63342/help/configure-modules.html#module-compiler-output) says:

> Select the Exclude output path checkbox to exclude the output folders from code completion, navigation, and inspections. This helps increase overall IDE performance.

**Comment by keturn** [comment]
Ow my head. I am learning more about this than I ever wanted to know.

One reason there are gradle/intellij fights is because gradle's java builder has [separate directories for class output and resource output](https://docs.gradle.org/current/userguide/java_plugin.html#sec:source_set_properties) but IntelliJ's combine both into one output directory. That's the reason why there are directories on the classpath named `build/resources` when running from gradle, but not IntelliJ.

I think, if those notes I wrote last night did work, what they must have been doing is telling IntelliJ to take the output from gradle's build and ... no, I'm not sure why / how that worked.

**Comment by keturn** [comment]
fixed by #3954
